### PR TITLE
*: adding trace syntax support

### PR DESCRIPTION
**Pull request by zhexuany**

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -2122,6 +2125,16 @@ EmptyStmt:
...
 	}
 
+TraceStmt:
+	"TRACE" TraceableStmt
+	{
+		$$ = &ast.TraceStmt{
+			Stmt:	$2,
+			Format: "row",
+		}
+
```
Comment (coocood) [inline parser/parser.y]: redundant empty line?

**Comment by coocood** [comment]
@zhexuany 
Please fix CI and add parser test.

**Comment by zhexuany** [comment]
@coocood I am currently working on this pr. I'll ping you once PR is ready to review. ;)

File: parser/parser.y [hunk 2]
```diff
@@ -5486,12 +5498,21 @@ Statement:
...
+|	TraceStmt
 |	TruncateTableStmt
 |	UpdateStmt
 |	UseStmt
 |	UnlockTablesStmt
 |	LockTablesStmt
 
+TraceableStmt:
+	SelectStmt
+|	DeleteFromStmt
```
Comment (zhexuany) [inline parser/parser.y]: I feel like only enable `SelectStmt` first and then support others? Any idea?

Comment (coocood) [reply parser/parser.y]: This question seems not related to this PR.

**Comment by coocood** [comment]
@zhexuany 
Please add parser tests

File: parser/parser.y (hunk 2 above)
Comment (zhexuany) [reply parser/parser.y]: `SelectStmt` need to be taken care of immediately. Other kinds of queries is not that important.

**Comment by coocood** [comment]
LGTM

File: parser/parser.y [hunk 3]
```diff
@@ -2129,6 +2132,15 @@ EmptyStmt:
 		$$ = nil
 	}
 
+TraceStmt:
+	"TRACE" TraceableStmt
+	{
+		$$ = &ast.TraceStmt{
+			Stmt:	$2,
+			Format: "row",
```
Comment (zimulala) [inline parser/parser.y]: Cloud we define it as a variable？

Comment (zhexuany) [reply parser/parser.y]: Just be consistent with `ExplainStmt`.

**Comment by zhexuany** [comment]
@zimulala Please take a look again, ;)

**Pull request merged**